app/controllers/home_controller.rb

```diff
@@ -10,4 +10,8 @@ def comingsoon
   def event
     render layout: "dayof"
   end
+
+  def apilist
+    render layout: "api"
```

Comment:
It looks like this is the same exact page style as the homepage - why the new layout?